packages/opencensus-core/src/tags/propagation/binary-serializer.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// import {TextDecoder} from 'util';
+
+import {VarInt} from '../../internal/util';
+import {TagMap} from '../tag-map';
+import {TagKey, TagValue} from '../types';
+
+export let VERSION_ID = 0;
+export let TAG_FIELD_ID = 0;
+// This size limit only applies to the bytes representing tag keys and values.
+const TAG_MAP_SERIALIZED_SIZE_LIMIT = 0;
```

Comment:
Why is this 0? In Java it's [8192](https://github.com/census-instrumentation/opencensus-java/blob/5a97afdefdb2d9f4213264d91fa67b646adbf5d9/impl_core/src/main/java/io/opencensus/implcore/tags/propagation/SerializationUtils.java#L72).